Add `o!BAM 2` Tournament (#7960)
ppy/osu-wiki: wiki/Tournaments/o!BAM/2/en.md

---
tags:
  - o!BAM
  - o!BAM 2
---

# osu! Battle Archive Malaysia 2

![o!BAM 2 banner](img/banner.png)

> The **osu! Battle Archive Malaysia 2** (***o!BAM2***) is a double-elimination 2v2 osu! tournament hosted by ::{ flag=MY }:: [LouisZpro](https://osu.ppy.sh/users/6231292) and ::{ flag=MY }:: [Tzero](https://osu.ppy.sh/users/6088976). The tournament was open to teams of two with a ::{ flag=MY }:: Malaysian player as their captain. Second player is only allowed from the ten [ASEAN member states](https://asean.org/about-asean/member-states/) (::{ flag=BN }:: Brunei Darussalam, ::{ flag=KH }:: Cambodia, ::{ flag=ID }:: Indonesia, ::{ flag=LA }:: Laos, ::{ flag=MY }:: Malaysia, ::{ flag=MM }:: Myanmar, ::{ flag=PH }:: The Philippines, ::{ flag=SG }:: Singapore, ::{ flag=TH }:: Thailand, and ::{ flag=VN }:: Vietnam) regardless of rank. Despite the change in format from its previous iterations, the tournament was formally regarded as the second instalment of the osu! Battle Archive Malaysia.

Reviewer: ```suggestion
The **osu! Battle Archive Malaysia 2** (***o!BAM2***) is a double-elimination 2v2 osu! tournament hosted by ::{ flag=MY }:: [LouisZpro](https://osu.ppy.sh/users/6231292) and ::{ flag=MY }:: [Tzero](https://osu.ppy.sh/users/6088976). The tournament is open to teams of two with a ::{ flag=MY }:: Malaysian player as their captain. Second player is only allowed from the ten [ASEAN member states](https://asean.org/about-asean/member-states/) (::{ flag=BN }:: Brunei Darussalam, ::{ flag=KH }:: Cambodia, ::{ flag=ID }:: Indonesia, ::{ flag=LA }:: Laos, ::{ flag=MY }:: Malaysia, ::{ flag=MM }:: Myanmar, ::{ flag=PH }:: The Philippines, ::{ flag=SG }:: Singapore, ::{ flag=TH }:: Thailand, and ::{ flag=VN }:: Vietnam) regardless of rank. Despite the change in format from its previous iterations, the tournament is formally regarded as the second instalment of the osu! Battle Archive Malaysia.
```

> The **osu! Battle Archive Malaysia 2022** (***o!BAM2***) is a double-elimination 2v2 osu! tournament hosted by ::{ flag=MY }:: [LouisZpro](https://osu.ppy.sh/users/6231292) and ::{ flag=MY }:: [Tzero](https://osu.ppy.sh/users/6088976). The tournament is open to teams of two with a ::{ flag=MY }:: Malaysian player as their captain. Second player is only allowed from the ten [ASEAN member states](https://asean.org/about-asean/member-states/) (::{ flag=BN }:: Brunei Darussalam, ::{ flag=KH }:: Cambodia, ::{ flag=ID }:: Indonesia, ::{ flag=LA }:: Laos, ::{ flag=MY }:: Malaysia, ::{ flag=MM }:: Myanmar, ::{ flag=PH }:: The Philippines, ::{ flag=SG }:: Singapore, ::{ flag=TH }:: Thailand, and ::{ flag=VN }:: Vietnam) regardless of rank. Despite the change in format from its previous iterations, the tournament is formally regarded as the second instalment of the osu! Battle Archive Malaysia.

Reviewer 2: ```suggestion
The **osu! Battle Archive Malaysia 2022** (***o!BAM2***) is a double-elimination 2v2 osu! tournament hosted by ::{ flag=MY }:: [LouisZpro](https://osu.ppy.sh/users/6231292) and ::{ flag=MY }:: [Tzero](https://osu.ppy.sh/users/6088976). The tournament is open to teams of two with a ::{ flag=MY }:: Malaysian player as their captain, and the second player coming from one of the ten [ASEAN member states](https://asean.org/about-asean/member-states/) (::{ flag=BN }:: Brunei Darussalam, ::{ flag=KH }:: Cambodia, ::{ flag=ID }:: Indonesia, ::{ flag=LA }:: Laos, ::{ flag=MY }:: Malaysia, ::{ flag=MM }:: Myanmar, ::{ flag=PH }:: The Philippines, ::{ flag=SG }:: Singapore, ::{ flag=TH }:: Thailand, and ::{ flag=VN }:: Vietnam) regardless of rank. Despite the change in format from its previous iterations, the tournament is formally regarded as the second instalment of the osu! Battle Archive Malaysia.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2022-08-19/2022-09-04 |
| Qualifiers | 2022-09-16/2022-09-18 |

> | Live drawings | 2022-09-20 ({22:00 UTC+8}) |

Reviewer: ```suggestion
| Live drawings | 2022-09-20 (22:00 UTC+8) |
```

| Group stage | 2022-09-23/2022-10-02 |
| Round of 16 | 2022-10-08/2022-10-09 |
| Quarterfinals | 2022-10-15/2022-10-16 |
| Semifinals | 2022-10-22/2022-10-23 |
| Finals week 1 | 2022-10-29/2022-10-30 |
| Finals week 2 | 2022-11-05/2022-11-06 |

## Organisation

> The osu! Battle Archive Malaysia 2 was run by various community members.

Reviewer: ```suggestion
The osu! Battle Archive Malaysia 2 is run by various community members.
```

  3. [BlackY feat. Risa Yuzuki - Shumatsu kinema (Scyhei) \[Reflection\]](https://osu.ppy.sh/beatmapsets/1648707#osu/3365231)
  4. [Kozakai Yurie, Omori Nichika, Hasegawa Rena - Hyper Bass (feat. Yunomi) (Hikan) \[ii\]](https://osu.ppy.sh/beatmapsets/1618954#osu/3305493)
- Hidden
  1. [Sota Fujimori - polygon (Sebu) \[-PC's hexagon\]](https://osu.ppy.sh/beatmapsets/422136#osu/1028537)
- Hidden + HardRock
  1. [sasanomaly - Onomatopoeia Glasses (Sagu) \[o-o-.\]](https://osu.ppy.sh/beatmapsets/1439058#osu/2960994)
- HardRock
  1. [Yamamoto Mineko - Cadena (Kazato Asa) \[Meruru Collab\]](https://osu.ppy.sh/beatmapsets/1214274#osu/2527237)
  2. [Miyo Takashiro feat. Aitsuki Nakuru - mirror (Avace) \[Celine's mirror\]](https://osu.ppy.sh/beatmapsets/1782658#osu/3650730)
- DoubleTime
  1. [Haloweak - Narwhal (meiikyuu) \[Rosetta\]](https://osu.ppy.sh/beatmapsets/1173876#osu/2474826)
  2. [ARM x Komso feat. Nanahira - Onegai! Kon kon Oinarisama ([Paw]OchiKouri) \[Muz's Insane\]](https://osu.ppy.sh/beatmapsets/1261912#osu/2635712)
  3. [Suara - Nuedori (VINXIS) \[emilia's Heavy Insane\]](https://osu.ppy.sh/beatmapsets/1143697#osu/2479193)

## Ruleset

> ### General

Reviewer: ```suggestion
### General rules
```
Maybe just for consistency

1. osu! Battle Archive Malaysia uses a double-elimination bracket and is a 2v2 osu! tournament.
2. Teams can only have 2 players. This means no substitution.
3. The score system used will be **Score V2**.
4. There is no solo rank range. A team rank range using the Badge Weighting System (BWS) is used instead with the following formula: `pp+(1- (rank**(0.9937**(badges**1.75))/rank))*pp`.
5. The 2 players must have an **average BWS PP of 12000pp**.
6. Teams will be seeded by **Qualifier seeding**.
7. Grand Finals will have a **bracket reset** if the team coming from loser's bracket wins the first match.
8. Match schedules and mappools **will be released on the Monday** before those matches should be played.
9. Match schedules will be written in UTC+8.
10. There will be pickems available **internationally** for anyone to have a chance at earning a prize, even if they aren't local.

### Round-specific rules

> #### Qualifier

Reviewer: ```suggestion
#### Qualifiers
```
Since you use Qualifiers in the timeline above
